### PR TITLE
testing/wireguard: various small fixes

### PR DESCRIPTION
**Pull request by itoffshore**
* fixes `$install_if` so the kernel modules install automatically when adding `wireguard-tools`
* test subpkg removed at the [request of the author](https://github.com/alpinelinux/aports/pull/948)
* `make check` removed & `sparse` removed from `makedepends` at the [request of the author](https://github.com/alpinelinux/aports/pull/948)

### CODE REVIEW

**Comment by itoffshore** [comment]
note to @zx2c4

File: testing/wireguard-tools/APKBUILD [hunk 1]
```diff
@@ -45,7 +43,7 @@ bashcomp() {
 }
 
 tests() {
```
Comment (zx2c4) [inline testing/wireguard-tools/APKBUILD]: This "tests package" simply _will not work_ out of context like this. It does not belong in a distribution package, no matter what it's called. If misunderstandings like this become more widespread than just you, then I wouldn't hesitate to just remove the code from the repository. It really doesn't belong.

Comment (itoffshore) [inline testing/wireguard-tools/APKBUILD]: test subpkg completely removed

Comment (zx2c4) [inline testing/wireguard-tools/APKBUILD]: Great, thanks!

**Comment by zx2c4** [comment]
Latest commit looks good to me. Please merge!

**Pull request closed without merging**